spec/controllers/person/tags_controller_spec.rb

```diff
@@ -166,6 +171,20 @@
       expect(bottom_member.tags.count).to eq(0)
       is_expected.to redirect_to group_person_path(bottom_member.groups.first, bottom_member)
     end
+
+    it "does delete tag if no other person uses it" do
```

Comment:
Why is the spec called "does delete tag ..." and your explanation is that id doesn't delete it with our option? Also this spec just checks if the taggable on the person (`expect(top_leader.tags.count).to eq(0)`) was destroyed, not if the tag still exists or not. The association may not exist anymore, but you never check if a tag was actually destroyed/not destroyed according to your explanation in this PR.